### PR TITLE
 testing MeasurementsForm component

### PR DESCRIPTION
**Pull request by code-knackerin**

### CODE REVIEW

File: src/components/MeasurementsForm.js [hunk 1]
```diff
@@ -37,12 +37,19 @@ const MeasurementsForm = (props) => {
     props.onAdd(measurementsInput);
   };
 
+  console.log();
```
Comment (domnikl) [inline src/components/MeasurementsForm.js:40]: can be removed safely 😉

File: src/components/MeasurementsForm.test.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  test("whether the type field is there", () => {
+    render(<MeasurementsForm />);
+
+    expect(screen.getAllByRole("option", { value: "Electricity" }).length).toBe(
+      3
+    );
+    expect(screen.getAllByRole("option", { value: "Gas" }).length).toBe(3);
+    expect(screen.getAllByRole("option", { value: "Water" }).length).toBe(3);
+    // const selectEl = screen.getByTestId("selectInputType", { exact: true });
```
Comment (domnikl) [inline src/components/MeasurementsForm.test.js:35]: Please remove code commented out

File: src/components/MeasurementsTable.js [hunk 3]
```diff
@@ -34,13 +34,14 @@ const MeasurementsTable = (props) => {
...
   const sortReadings = filteredReadings.sort(
-    (a, b) => new Date(b.date) - new Date(a.date)
+    (a, b) => new Date(b.date) - new Date(a.date) 
   );
   console.log(filteredReadings);
 
   const measurementsHandleChange = (measurement) => {
     setMeasurementsList([...measurementsList, measurement]);
   };
+  //id hinzufügen - neues object mit alten werten und Id als neue dazu
```
Comment (domnikl) [inline src/components/MeasurementsTable.js:44]: please remove comment not necessary

**Comment by code-knackerin** [comment]
Thanks! It's done

**Pull request merged**